src/Features/CSharp/Portable/Completion/CompletionProviders/CompletionUtilities.cs

```diff
@@ -96,31 +96,16 @@ internal static bool IsTriggerCharacter(SourceText text, int characterPosition,
         /// </summary>
         internal static bool IsCompilerDirectiveTriggerCharacter(SourceText text, int characterPosition)
         {
-            // First we check if current character is space
-            var ch = text[characterPosition--];
-            if (ch != ' ')
-            {
-                return false;
-            }
-
-            // Then step back while "current" character is letter
-            while (char.IsLetter(text[characterPosition]))
+            while (char.IsWhiteSpace(text[characterPosition]) ||
```

Comment:
I was concerned specifically about tabs, not all whitespace characters.
We now check new lines, so it's possible to go too far beyond the current line.

Reply:
I thought, it is just a wrapper for a `ch == ' '` check... Ok, reverting